workspaces/ui-v2/src/types/endpoints.ts

```diff
@@ -0,0 +1,17 @@
+export interface IPathParameter {
+  id: string;
+  name: string;
+  isParameterized: boolean;
+  description: string;
+  endpointId: string;
+}
+
+export interface IEndpoint {
+  pathId: string;
+  method: string;
+  purpose: string;
+  description: string;
+  fullPath: string;
+  pathParameters: IPathParameter[];
+  group: string;
+}
```

Comment:
I pulled out `changelog` from this interface, because it is only used in one place and should be fetched separately